ioctl/newcmd/contract/contractdeploy.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2022 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
```

Comment:
remove stale comments: `This is an alpha (internal) release and is not suitable for production.`